src/test/scala/io/iohk/ethereum/mpt/MerklePatriciaTrieSuite.scala

```diff
@@ -11,12 +12,19 @@ import io.iohk.ethereum.mpt.MerklePatriciaTrie.{MPTException, defaultByteArraySe
 import org.scalacheck.{Arbitrary, Gen}
 import org.bouncycastle.util.encoders.Hex
 import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+
 import scala.util.{Random, Try}
 import org.scalatest.funsuite.AnyFunSuite
 
 class MerklePatriciaTrieSuite extends AnyFunSuite with ScalaCheckPropertyChecks with ObjectGenerators {
 
-  val EmptyEphemNodeStorage = StateStorage.createTestStateStorage(EphemDataSource())._1.getBackingStorage(0)
+  val source: EphemDataSource = EphemDataSource()
+  val tuple: (StateStorage, NodeStorage, CachedNodeStorage) = StateStorage.createTestStateStorage(source)
+  val EmptyEphemNodeStorage = {
+    val value: StateStorage = tuple._1
```

Comment:
Maybe we can use this

val (stateStorage, nodeStorage, achedNodeStorage): (StateStorage, NodeStorage, CachedNodeStorage) = StateStorage.createTestStateStorage(source)
  val EmptyEphemNodeStorage = {
    val value: StateStorage = stateStorage
    value.getBackingStorage(0)
  }
  val EmptyNodeStorage = nodeStorage

Reply:
Nice idea, I end up with:
```scala
  val (stateStorage, emptyNodeStorage, _) = StateStorage.createTestStateStorage(dataSource)
  val emptyEphemNodeStorage = stateStorage.getBackingStorage(0)
```
good enough?